.github/workflows/run-tests.yml

```diff
@@ -63,7 +71,7 @@ jobs:
           cd ${{ env.REMOTE_WORK_DIR }}
           conda env create -f ./conda-reqs.yaml -p ./.conda-env
           eval "$(conda shell.bash hook)"
-          conda install -y -p $PWD/.conda-env -c ucb-bar riscv-tools
+          conda install -y -p $PWD/.conda-env -c ucb-bar riscv-tools=1.0.4
```

Comment:
Does this need to be updated in the CY conda env?

Reply:
1.0.3 and 1.0.4 should be near equivalent.